modules/apps/analytics/analytics-settings-impl/src/main/java/com/liferay/analytics/settings/internal/configuration/AnalyticsConfigurationRegistryImpl.java

```diff
@@ -129,18 +128,15 @@ public Dictionary<String, Object> getAnalyticsConfigurationProperties(
 			return null;
 		}
 
-		Set<Map.Entry<String, Long>> entries = _companyIds.entrySet();
+		String pid = null;
 
-		Stream<Map.Entry<String, Long>> stream = entries.stream();
+		for (Map.Entry<String, Long> entry : _companyIds.entrySet()) {
+			if (Objects.equals(entry.getValue(), companyId)) {
+				pid = entry.getKey();
 
-		String pid = stream.filter(
-			entry -> Objects.equals(entry.getValue(), companyId)
-		).map(
-			Map.Entry::getKey
-		).findFirst(
-		).orElse(
-			null
-		);
+				break;
+			}
+		}
```

Comment:
@julschong Please merge the following part into the for loop to avoid multiple return points

Reply:
resubmitted here: https://github.com/liferay-core-infra/liferay-portal/pull/3299